scripts/install.py

```diff
@@ -1465,6 +1516,47 @@ def tweak_malcolm_runtime(self, malcolm_install_path):
                 'ARKIME_AUTO_ANALYZE_PCAP_FILES',
                 TrueOrFalseNoQuote(autoArkime),
             ),
+            # Should Arkime use an ILM policy?
+            EnvValue(
```

Comment:
rather than sticking the booleans into the .env files directly, use `TrueOrFalseNoQuote(ilmPolicy)` and `TrueOrFalseNoQuote(ilmHotWarm)` to do it the standard way like the other booleans are written to their .env files. As you're doing it here what you'd end up with is `True` and `False` in the .env file which is not what we're checking for in the bash script.